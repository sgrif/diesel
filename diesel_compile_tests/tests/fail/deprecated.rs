#[macro_use]
extern crate diesel;

table! {
    users {
        id -> Integer,
        name -> Text,
    }
}

#[derive(Identifiable)]
#[table_name = "users"]
#[primary_key(id)]
struct UserForm {
    id: i32,
    #[column_name = "name"]
    name: String,
}

fn main() {
    // Workaround for https://github.com/dtolnay/trybuild/issues/8
    compile_error!("warnings");
}
