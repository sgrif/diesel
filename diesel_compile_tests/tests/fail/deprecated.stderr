warning: #[table_name] attribute form is deprecated
  --> $DIR/deprecated.rs:12:3
   |
12 | #[table_name = "users"]
   |   ^^^^^^^^^^

error: expected identifier
  --> $DIR/deprecated.rs:12:16
   |
12 | #[table_name = "users"]
   |                ^^^^^^^

error: warnings
  --> $DIR/deprecated.rs:22:5
   |
22 |     compile_error!("warnings");
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
